app/tasks/usage.php

```diff
@@ -0,0 +1,373 @@
+<?php
+
+global $cli, $register;
+
+require_once __DIR__ . '/../init.php';
+
+use Utopia\App;
+use Utopia\Cache\Adapter\Redis;
+use Utopia\Cache\Cache;
+use Utopia\CLI\Console;
+use Utopia\Database\Adapter\MariaDB;
+use Utopia\Database\Database;
+use Utopia\Database\Document;
+use Utopia\Database\Query;
+use Utopia\Database\Validator\Authorization;
+
+/**
+ * Metrics We collect
+ *
+ * requests
+ * network
+ * executions
+ * database.collections.create
+ * database.collections.read
+ * database.collections.update
+ * database.collections.delete
+ * database.documents.create
+ * database.documents.read
+ * database.documents.update
+ * database.documents.delete
+ * database.collections.{collectionId}.documents.create
+ * database.collections.{collectionId}.documents.read
+ * database.collections.{collectionId}.documents.update
+ * database.collections.{collectionId}.documents.delete
+ * storage.buckets.{bucketId}.files.create
+ * storage.buckets.{bucketId}.files.read
+ * storage.buckets.{bucketId}.files.update
+ * storage.buckets.{bucketId}.files.delete
+ * users.create
+ * users.read
+ * users.update
+ * users.delete
+ * users.sessions.create
+ * users.sessions.delete
+ *
+ * Counters
+ *
+ * users.count
+ * storage.files.count
+ * database.collections.count
+ * database.documents.count
+ * database.collections.{collectionId}.documents.count
+ *
+ * Totals
+ *
+ * storage.total
+ *
+ */
+
+$cli
+    ->task('usage')
+    ->desc('Schedules syncing data from influxdb to Appwrite console db')
+    ->action(function () use ($register) {
+        Console::title('Usage Aggregation V1');
+        Console::success(APP_NAME . ' usage aggregation process v1 has started');
+
+        $interval = (int) App::getEnv('_APP_USAGE_AGGREGATION_INTERVAL', '30'); //30 seconds
+        $periods = [
+            [
+                'key' => '30m',
+                'startTime' => '-24 hours',
+            ],
```

Comment:
Are we missing the 30 days period here ? @lohanidamodar

Reply:
Nope, for 30 days we get `1d` period

Reply:
Alright :+1: